swatch-system-conduit/src/main/java/org/candlepin/subscriptions/conduit/InventoryController.java

```diff
@@ -65,9 +59,13 @@
   private static final BigDecimal KIBIBYTES_PER_GIBIBYTE = BigDecimal.valueOf(1048576);
   private static final BigDecimal BYTES_PER_KIBIBYTE = BigDecimal.valueOf(1024);
   private static final String COMMA_REGEX = ",\\s*";
+  private static final String PERIOD_REGEX = "\\.";
   private static final String NON_HYPHEN_REGEX = "[0-9a-fA-F]{8}([0-9a-fA-F]{4}){3}[0-9a-fA-F]{12}";
   private static final String UUID_REGEX = "[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}";
 
+  public static final String OS_DISTRIBUTION_NAME = "distribution.name";
+  public static final String DISTRIBUTION_VERSION = "distribution.version";
```

Comment:
nitpick: for consistency:
```suggestion
  public static final String OS_DISTRIBUTION_VERSION = "distribution.version";
```